2025-07-07 19:39:34,339:ERROR:search-app2:Item Chris not found in the list. Error: 'Chris' is not in list
Traceback (most recent call last):
  File "C:\Users\mario\OneDrive\Υπολογιστής\Python-CF7\chapter05\18_logging\18_logging_extra.py", line 84, in search_items
    index = items.index(item_to_find)
ValueError: 'Chris' is not in list
2025-07-07 19:39:58,848:INFO:search-app2:Item Bob found at index 1
